mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java

```diff
@@ -65,6 +69,20 @@ public void initViews() {
 
     }
 
+    public void initExpand() {
+        final int[] expand_pos = {-1};
+        expandableListView.setOnGroupExpandListener(
+                new ExpandableListView.OnGroupExpandListener() {
+                    @Override
+                    public void onGroupExpand(int groupPosition) {
+                        if (expand_pos[0] >= 0 && expand_pos[0] != groupPosition) {
+                            expandableListView.collapseGroup(expand_pos[0]);
+                        }
+                        expand_pos[0] = groupPosition;
```

Comment:
"Cannot assign a value to final variable 'expand_pos'" showing this error when I convert it into normal int

Reply:
@luckyman20

Reply:
what I meant was `final int expand_pos = -1`

Reply:
Yes when I do that, 

> "Cannot assign a value to final variable 'expand_pos'" showing this error when I convert it into normal int

this error is coming in the end of the function

Reply:
![image](https://user-images.githubusercontent.com/37215508/71641373-e07f3f00-2cc0-11ea-9950-8fc1aac07a3e.png)

Reply:
Remove `final` keyword.

Reply:
Tried that too sir but `Variable 'expand_pos' is accessed from within inner class, needs to be declared final` error occurs :(

Reply:
This issue is solved if I add int expand_pos over here. Please have a look
![image](https://user-images.githubusercontent.com/37215508/71641466-664fba00-2cc2-11ea-976e-f875ff8e9c00.png)

Reply:
This one works. :+1: